### PR TITLE
[feature] sample project  and pre commit

### PR DESCRIPTION
**Pull request by a-c-caeiro**
sample project
add pre commit

### CODE REVIEW

File: docs/pre-commit.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    env
+    .env
+    ./env
+    env/
+    .env/
+    .venv/
+    inspectdb_models
+```
+
+## Activation
```
Comment (kenvontucky) [inline docs/pre-commit.md]: This heading should be activation / deactivation

**Review by kenvontucky — CHANGES_REQUESTED** [review]
Can we fill in the gap between the sample_django_project and the documentation, like how to use the Makefile? We could simply add a Readme file to the root of the sample django project, what do you think?

File: docs/pre-commit.md (hunk 1 above)
Comment (antonio-caeiro) [inline docs/pre-commit.md]:
```suggestion
## Activation / Deactivation
```

**Review by kenvontucky — APPROVED** [review]

File: sample_django_project/.env.sample [hunk 2]
```diff
@@ -0,0 +1,6 @@
+SECRET_KEY="very_secret_much_security_such_wow_black_mamba_12345"
```
Comment (brunofilipe) [inline sample_django_project/.env.sample:1]: 🚀

**Review by brunofilipe — APPROVED** [review]

**Pull request merged**